### PR TITLE
Preparing the AAP Operator  guides for EA

### PR DESCRIPTION
**Pull request by g-murray**
[AAP-20941](https://issues.redhat.com/browse/AAP-20941) - Install gateway
[AAP-22178](https://issues.redhat.com/browse/AAP-22718) - Gateway Backup and restore sections

Preparing operator docs ahead of the 2.5 EA release

Docs:
-  **Red Hat Ansible Automation Platform operator backup and recovery guide**
-  **Installing on OpenShift Container Platform** 
(formally known as the Deploying the Red Hat Ansible Automation Platform operator on OpenShift Container Platform) 

Locations:
Doc location: Downstream > titles > aap-operator-installation
Doc location Downstream > titles > aap-operator-backup

Updates:
- Updating existing chapters to allign ahead of the new Gateway content.
- Uncommenting the module include statements for gateway install
- Uncommenting module include statements for backup and restore

### CODE REVIEW

File: downstream/modules/platform/con-ocp-supported-install.adoc [hunk 1]
```diff
@@ -7,9 +7,30 @@ You can use the OperatorHub on the {OCP} web console to install {OperatorPlatfor
...
-* {HubNameStart} custom resources first, then {ControllerName} custom resources;
-* {ControllerNameStart} custom resources;
-* {HubNameStart} custom resources.
+If existing components have already been deployed, you must specify these components on the {PlatformNameShort} CR. You must create the custom resource in the same namespace as the existing components.
+
+[cols=2*a,options="header"]
+|===
+| *Supported scenarios* | *Supported scenarios with existing components*
+|
+* {PlatformNameShort} CR for blank slate install with automation controller, automation hub, and EDA enabled
```
Comment (michellemacrh) [inline downstream/modules/platform/con-ocp-supported-install.adoc:18]: Just a suggestion but you could use attributes for automation controller, hub and EDA 👍

File: downstream/modules/platform/proc-operator-external-db-hub.adoc [hunk 2]
```diff
@@ -22,7 +22,7 @@ The external database must be a PostgreSQL database that is the version supporte
 
 [NOTE]
 ====
-{PlatformNameShort} {PlatformVers} supports PostgreSQL 13.
+{PlatformNameShort} {PlatformVers} supports PostgreSQL 15.
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-operator-external-db-hub.adoc:25]: PostgreSQL 15 could be replaced with `{PostgresVers}`

**Review by michellemacrh — COMMENTED** [review]
2 small suggestions otherwise LGTM!

File: downstream/modules/platform/con-ocp-supported-install.adoc (hunk 1 above)
Comment (g-murray) [reply downstream/modules/platform/con-ocp-supported-install.adoc]: Done!

File: downstream/modules/platform/proc-operator-external-db-hub.adoc (hunk 2 above)
Comment (g-murray) [reply downstream/modules/platform/proc-operator-external-db-hub.adoc]: Done!

File: downstream/modules/platform/con-ocp-supported-install.adoc [hunk 3]
```diff
@@ -7,29 +7,29 @@ You can use the OperatorHub on the {OCP} web console to install {OperatorPlatfor
 
 Alternatively, you can install {OperatorPlatform} from the {OCPShort} command-line interface (CLI), `oc`.
 
-After you have installed {OperatorPlatform} you must create an *AnsibleAutomationPlatform* custom resource (CR). This allows you to manage {PlatformNameShort} components from a single unified interface known as the platform gateway. As of version 2.5, you must create an {PlatformNameShort} CR, even if you have an existing automation controller,  automation hub, or Event-driven Ansible (EDA), components.
+After you have installed {OperatorPlatform} you must create an *AnsibleAutomationPlatform* custom resource (CR). This allows you to manage {PlatformNameShort} components from a single unified interface known as the platform gateway. As of version 2.5, you must create an {PlatformNameShort} CR, even if you have an existing {ControllerName},  {HubName}, or {{EDAName}}, components.
```
Comment (michellemacrh) [inline downstream/modules/platform/con-ocp-supported-install.adoc:10]: Looks like `{{EDAName}}` has an extra set of curly brackets

Comment (g-murray) [reply downstream/modules/platform/con-ocp-supported-install.adoc]: Fixed!

**Comment by michellemacrh** [comment]
LGTM

**Pull request merged**